### DISCUSSION TITLE
How can I prevent a Hydration failed error in Nextjs when opening a pdf file from a link that is in a translated route?

### DISCUSSION THREAD

**Discussion by crazyyi** [lingui/js-lingui#1513] [Q&A]
To make it clear, I have a Link on my page say:
```
<Link target={"_blank"} href={"/documents/open-me.pdf"}>Click Me</Link>
```
It works great in defaultLocale. Everything renders correctly on the page, and when I click the link it opens the pdf file which shows the route "http://localhost:3000/documents/open-me.pdf"
But then if I switch to another locale, say "fr", and then I go to the page again, click the link, it goes to the route "http://localhost:3000/fr/documents/open-me.pdf", but this time the pdf cannot be opened, and it gives me this error:
```
Unhandled Runtime Error
Error: Hydration failed because the initial UI does not match what was rendered on the server.
```
Since I can't do anything on the pdf file, what can I do to prevent this error when users want to open the file?

**Comment by timofei-iatsenko** [comment]
Look for the issue in the app code. Nextjs is weird, issue might be in other place which you are not expecting. Nothing to do from Lingui side.
